services/checks/checks.go

```diff
@@ -521,6 +527,17 @@ func (s *Service) findTargets(serviceType models.ServiceType) ([]target, error)
 	return targets, nil
 }
 
+// isSystem returns true if provided service is system
+func isSystem(service *models.Service) bool {
```

Comment:
We skip them because we control everything inside PMM Server container/VM, and don't want users to try to update them.
That reason might affect the name of the function ("isSystem" seems to be too generic? and we don't call them "system services" anywhere) and implementation (we really care about "service.NodeID == models.PMMServerNodeID" only)

@BupycHuk thoughts?

Reply:
@BupycHuk ^

Reply:
Why don't we just check if current service has name equal to `PMMPostgreSQLServiceName`? In that case we don't even need new method for that

Reply:
If such comparison is enough then ok. I tried to achieve a maximum match to eliminate bugs.

Reply:
@AlekSi @BupycHuk 
So we have three options:
1)Current function, that compares NodeID, ServiceType and ServiceName
2)Compare only NodeID
3)Compare only ServiceName

Service name seems to me like a weak condition. If we haven't any plans to run STT checks against any PMM server component(probably not), then comparison by NodeID is enough IMHO. Do you agree?

Reply:
Ok, let's try to check only NodeID